rfcs/create-new-test-editor-repo.md

```diff
@@ -0,0 +1,16 @@
+# RFC #24 - Add the Editor repo
+
+## Summary
+This repo allows developers to create and edit tests to be consumed by the WPT test infrastructure.
+
+## Details
+Originally, this repo was built by Microsoft to help facilitate debugging and authoring of tests for Web Platform Tests.
+
+It's been living under the MicrosoftEdge repo up until now, but we believe the community will benefit from this tool.
+
+By moving it to the Web-Platform-Tests repo, the people who are directly working with WPT will have access and context.
+
+The repo would live [here](https://github.com/web-platform-tests/editor).
+
+## Risks
+This repo was created by an ex-Microsoft employee. Microsoft's current expertise in this repo is limited, so there is some risk regarding maintenance.
```

Comment:
Do we have any idea of who will maintain it? It's not magically going to gain maintainers by just moving what GitHub org it's under. I'd rather not add it under the WPT org until we have someone saying they'll maintain it; adding it unmaintained is of dubious usefulness.

Reply:
I can promise at least some maintenance time on it.

Reply:
I will as well.

Reply:
I've added a sentence re me and Tab.